apps/channels/views.py

```diff
@@ -24,3 +25,21 @@ def new_whatsapp_message(request):
     message_data = json.dumps(request.POST.dict())
     tasks.handle_whatsapp_message.delay(message_data)
     return HttpResponse()
+
+
+@csrf_exempt
+def new_facebook_message(request: HttpRequest):
```

Comment:
Since this has to be setup for each bot, what do you think about adding a `team_slug` to the URL path which would allow us to further filter the channel query.

Reply:
Good idea!

Reply:
https://github.com/dimagi/open-chat-studio/pull/115/commits/44d16329299546cbcd99cab4dc2dcfcf654496b8